src/plugins/tabs/_screen-sm-max.scss

```diff
@@ -9,20 +9,29 @@
 %tabs-screen-sm-max-details {
 	> {
 		details {
-			border-width: $tablist-screen-sm-details-border-width;
-      border-style: $tablist-screen-sm-details-border-style;
-      border-color: $tablist-screen-sm-details-border-color;
-			border-top: 0;
+			border: {
+				width: $tablist-screen-sm-details-border-width;
+				style: $tablist-screen-sm-details-border-style;
+				color: $tablist-screen-sm-details-border-color;
+			}
 			padding: 6px 12px;
 
 			&:first-of-type {
-				border-top-left-radius: $tablist-screen-sm-details-border-radius;
-				border-top-right-radius: $tablist-screen-sm-details-border-radius;
+				border: {
+					top: {
+						left-radius: $tablist-screen-sm-details-border-radius;
+						right-radius: $tablist-screen-sm-details-border-radius;
+					}
```

Comment:
Need to optimize this so that it's not top or bottom otherwise it loose it's flexibility